action.yml

```diff
@@ -1,5 +1,5 @@
 # azure spirng cloud action
-name: 'AzureSpringCloud'
+name: 'Azure Spring Cloud'
```

Comment:
I understand this is because we have a conflict in name. 2 questions here:

1. why are we getting such conflict?
2. what's the naming practice in WebApp / Functions? We should follow the naming conventions from them.

Reply:
I am not sure why we get conflict now. I add space because WebApp named as 'Azure WebApp'